tests/api/v2/datadog/api_dashboard_lists_test.go

```diff
@@ -65,7 +65,9 @@ func deleteDashboardList() {
 }
 
 func TestDashboardListItemCRUD(t *testing.T) {
-	ctx, finish := WithRecorder(WithTestAuth(context.Background()), t)
+	ctx, finish := tests.WithTestSpan(context.Background(), t)
+	defer finish()
+	ctx, finish = WithRecorder(WithTestAuth(ctx), t)
 	defer finish()
```

Comment:
Not super familiar, but does the double `defer` with the same method name work as expected here?

Reply:
AFAIK there is no name binding, so it's not a problem. At least recording works fine, we can check if spans are reported properly.

Reply:
Ok 👍 Thanks